NOTES.md

```diff
@@ -0,0 +1,28 @@
+## Set up
+
+```sh
+wget https://dlcdn.apache.org/spark/spark-3.4.1/spark-3.4.1-bin-hadoop3.tgz
+mkdir spark-bin
+tar -xvf spark-3.4.1-bin-hadoop3.tgz --directory spark-bin
+./spark-bin/spark-3.4.1-bin-hadoop3/sbin/start-connect-server.sh --packages org.apache.spark:spark-connect_2.12:3.4.0
```

Comment:
Is this the only way to launch a spark connect server?

Reply:
for local test, we can use `bin/pyspark --remote local` to start connect server locally and start a spark python REPL at the same time